src/Apps/Sell/Routes/AdditionalRoutes/ShippingLocationRoute.tsx

```diff
@@ -154,9 +153,6 @@ export const ShippingLocationRoute: React.FC<ShippingLocationRouteProps> = props
                       value={values.location.address}
                       onChange={handleChange}
                       onBlur={handleBlur}
-                      error={
-                        touched.location?.address && errors.location?.address
-                      }
```

Comment:
We don't want to display the error message because we disable the submit button instead.